app/src/main/java/com/example/myweatherapp/utils/WeatherUtils.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.myweatherapp.utils
+
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+const val CITY = "Tandil"
+const val EMPTY_STRING = ""
+const val URL = "http://openweathermap.org/img/wn/"
```

Comment:
this URL and the FORMAT you can keep it in the adapter into a companion because it is used only there

Reply:
Done thx